src/pages/careers/_model/_locations/_locations.ts

```diff
@@ -522,6 +523,31 @@ export const georgetown: LocationsType = {
     has_map: true,
 }
 
+export const roadtown: LocationsType = {
+    name: 'roadtown',
+    job_location: 'British-Virgin-Islands',
+    continent: 'caribbean',
+    display_continent: 'Caribbean',
+    link: '/careers/locations/roadtown/',
+    display_name: 'Road Town',
+    display_thumbnail_name: 'Road Town',
+    display_title_name: 'Road Town',
+    img_alt: ' Deriv’s Cayman Islands office at Road Town ',
+    map_office_name: '(BVI)',
+    country: 'British Virgin Islands',
+    flagIcon: CaymanFlagIcon,
+    thumbnail: 'thumbnail_roadtown',
+    first_p:
+        'Experts in regulatory and legislative requirements will work together at Deriv (BVI) to provide compliance guidance in our British Virgin Islands licensed operations. Their expertise in corporate law, anti-money laundering, and other legislation will enable them to anticipate and mitigate regulatory risks. ',
+    first_img: 'living_in_roadtown',
+    first_img_alt: 'Deriv’s office in the Cayman Islands',
```

Comment:
ditto, this is from Cayman Islands office should be in respective of the BVI(Road Town) office